lib/Twig/Loader/Filesystem.php

```diff
@@ -285,7 +285,7 @@ private function isAbsolutePath($file)
     {
         return strspn($file, '/\\', 0, 1)
             || (strlen($file) > 3 && ctype_alpha($file[0])
-                && substr($file, 1, 1) === ':'
+                && ':' === $file[0]
```

Comment:
is something like appveyor for Windows test left out of the repo by design or because of lack of time to set it up?